setup.py

```diff
@@ -25,10 +25,10 @@
     install_requires=[
          'Flask-FeatureFlags==0.6',
          'Flask-Script==2.0.5',
-         'Flask-WTF==0.14.2',
-         'Flask>=0.10.1',
+         'Flask-WTF==0.12',
+         'Flask>=0.10',
```

Comment:
Why do we do `>=`. We have Flask pinned at exactly `0.10.1` in the apps.

Reply:
Again, it's an exact revert, so we can deal with these questions at a later date (in theory).